source/functionality/search.md

```diff
@@ -0,0 +1,35 @@
+# Enable search
+
+The search function is especially useful for multipage documentation.
+
+## Amend the tech-docs.yml file
+
+Add the following to the end of your project’s `.../config/tech-docs.yml` file, or set it to true if it is already there:
```

Comment:
Doesn't need to be at the end, in my experience.
This prompted me to look up whether order matters in YAML files, and [it doesn't matter for key-value pairs of the same indentation](https://github.com/Animosity/CraftIRC/wiki/Complete-idiot's-introduction-to-yaml).

Reply:
It doesn't indeed: I commented to the same effect in https://github.com/alphagov/tdt-documentation/pull/15